From: Andrea Zagli <azagli@libero.it>
Date: Sun, 22 May 2016 19:53:21 +0200
Subject: aclocal wheezy.

---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 83f5de2..9e5ebc4 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -22,7 +22,7 @@ if [ "$#" = 0 -a "x$NOCONFIGURE" = "x" ]; then
 fi
 
 set -x
-aclocal --install || exit 1
+aclocal --install -I m4 || exit 1
 glib-gettextize --force --copy || exit 1
 #gtkdocize --copy || exit 1
 intltoolize --force --copy --automake || exit 1
